includes/shortcodes/class-wc-shortcode-cart.php

```diff
@@ -66,10 +66,11 @@ public static function output( $atts ) {
 		// Constants.
 		wc_maybe_define_constant( 'WOOCOMMERCE_CART', true );
 
-		$atts = shortcode_atts( array(), $atts, 'woocommerce_cart' );
+		$atts        = shortcode_atts( array(), $atts, 'woocommerce_cart' );
+		$nonce_value = wc_get_var( $_REQUEST['woocommerce-shipping-calculator-nonce'], wc_get_var( $_REQUEST['_wpnonce'], '' ) ); // @codingStandardsIgnoreLine.
 
 		// Update Shipping.
-		if ( ! empty( $_POST['calc_shipping'] ) && ! empty( $_POST['_wpnonce'] ) && wp_verify_nonce( wc_clean( wp_unslash( $_POST['_wpnonce'] ) ), 'woocommerce-cart' ) ) { // WPCS: input var ok.
```

Comment:
Does this need some sort of backwards-compatibility handling because the nonce we're verifying changed. Specifically for themes that are using the old version of `templates/cart/shipping-calculator.php`?

Reply:
Yeah good catch. We can check for both strings.